### PR TITLE
Convert CacheVol regression tests into unit tests

### PR DESCRIPTION
**Pull request by masaori335**

### CODE REVIEW

File: src/iocore/cache/test/storage.config [hunk 1]
```diff
@@ -1,2 +1,2 @@
-var/trafficserver 32M
+var/trafficserver 256M
```
Comment (masaori335) [inline src/iocore/cache/test/storage.config:1]: If I set larger value than this, the test failed intermittently. I left the issue from this PR.
https://github.com/apache/trafficserver/issues/10648

**Review by bneradt — DISMISSED** [review]
Very nice. I did a diff of the old and new sections and they look functionally identical.

**Comment by masaori335** [comment]
Resolved conflicts, please take a look again.

**Comment by masaori335** [comment]
[approve ci autest]

**Review by bneradt — APPROVED** [review]

**Pull request merged**